db/db.js

```diff
@@ -22,13 +21,17 @@ if (process.env.DATABASE_URL) {
   };
 }
 
-const db = new Sequelize(`fit`, "postgres", "7aa7aa7a8a", {
-  host: "localhost",
-  dialect: "postgres",
+//we can add the .env to the gitignore and store the database password below (cna be different for each use
```

Comment:
I was thinking this. I just made my database password the same so it isn't an issue currently